### PR TITLE
Add timing and commit to the footer

### PR DESCRIPTION
**Pull request by KorvinSzanto**
This overrides the footer element from the theme replacing the uptime monitoring statement with the current commit hash and a middleware that reports how long rendering took.

### CODE REVIEW

File: public/application/bootstrap/app.php [hunk 1]
```diff
@@ -63,3 +63,12 @@
...
  */
+
+$class = \Concrete\Core\Http\ServerInterface::class;
+if ($app->resolved($class) && $server = $app->make($class)) {
+    $server->addMiddleware(new \Concrete5\Translate\Http\TimingMiddleware(), 100);
+} else {
+    $app->resolving($class, function(\Concrete\Core\Http\ServerInterface $server) {
+        $server->addMiddleware(new \Concrete5\Translate\Http\TimingMiddleware(), 100);
+    });
+}
```
Comment (mlocati) [inline public/application/bootstrap/app.php:74]: Smart!

File: public/application/elements/footer_navigation.php [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+                <div class="col-sm-2 text-center">
+                    <a href="http://laravel.com/" target="_blank">
+                        <img alt="Laravel gives us Config, Application Container, Filesystem, and more!"
+                             src="/packages/concrete5_theme/themes/concrete5/images/logos/laravel.png" />
+                    </a>
+                </div>
+            </div>
+        </div>
+    </div>
+    <? /*
```
Comment (mlocati) [inline public/application/elements/footer_navigation.php:46]: Do we allow short tags?

Comment (KorvinSzanto) [reply public/application/elements/footer_navigation.php:46]: this is a mostly verbatim copy from the theme dependency, so I'd hope so

File: public/application/elements/footer_navigation.php [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+                    </a>
+                        <span>
+                            <span>Copyright <?= date('Y') ?> </span>
+                            <a style="color:#666" href="http://www.portlandlabs.com/" target="_blank">Portland Labs</a>
+                            <span>&nbsp; &middot; &nbsp; All Rights Reserved</span>
+                        </span>
+                </div>
+                <div class="col-sm-6 social-icons">
+                    <div class='pull-right' style="color:#666">
+                        <span>Rev. <strong><?= substr(getenv('COMMIT') ?: 'DEV', 0, 6) ?></strong> rendered in <strong>{{ RENDER_TOTAL }}</strong></span>
```
Comment (mlocati) [inline public/application/elements/footer_navigation.php:168]: I'm very curious about how this `COMMIT` environment variable is set

**Pull request merged**